.github/config/regression-tests.yml

```diff
@@ -0,0 +1,802 @@
+            "forge-std/=lib/forge-std/src/"
+          ],
+          "settings": {
+            "optimizer": {
+              "runs": 9999999
+            }
+          }
+        },
+        "solidityTest": {
+          "testFail": true
+        }
+      };
+    ref: 21213d34042b8a5a68afeb590f43018f08c81a58
+  mds1/multicall:
+    forge-version: v0.3.0
+    hardhat-config: |
+      export default {
+        "paths": {
+          "sources": "src",
+          "tests": {
+            "solidity": "src/test"
+          }
+        },
+        "solidity": {
+          "version": "0.8.12",
+          "settings": {
+            "optimizer": {
+              "enabled": true,
+              "runs": 10000000
+            }
+          },
+          "remappings": [
+            "forge-std/=lib/forge-std/src/",
+            "ds-test/=lib/forge-std/lib/ds-test/src/"
+          ]
+        },
+        "solidityTest": {
+          "fuzz": {
+            "runs": 1000
+          }
+        }
+      };
+    ref: 19da7a56668b7b284901ef0551edb23905c6cd53
+  pancakeswap/infinity-core:
+    forge-version: v0.3.0
+    hardhat-config: |
+      export default {
+        "paths": {
+          "sources": "src",
+          "tests": {
+            "solidity": "test"
+          }
+        },
+        "solidity": {
+          "version": "0.8.26",
+          "remappings": [
+            "ds-test/=lib/forge-std/lib/ds-test/src/",
+            "forge-std/=lib/forge-std/src/",
+            "@openzeppelin/=lib/openzeppelin-contracts/",
+            "solmate/=lib/solmate/",
+            "forge-gas-snapshot/=lib/forge-gas-snapshot/src/"
+          ],
+          "settings": {
+            "viaIR": true,
+            "optimizer": {
+              "runs": 25666
+            },
+            "evmVersion": "cancun",
+            "metadata": {
+              "bytecodeHash": "none"
+            }
+          }
+        },
+        "solidityTest": {
+          "testFail": true,
+          "ffi": true,
+          "fsPermissions": {
+            "readWrite": [
+              ".forge-snapshots/"
+            ],
+            "read": [
+              "./foundry-out",
+              "./script/config",
+              "./test/pool-cl/bin",
+              "./test/pool-bin/bin"
+            ]
+          },
+          "blockGasLimit": BigInt(300000000),
+          "fuzz": {
+            "runs": 5
+          }
+        }
+      };
+    ignore: |
+      # TokenLocker.setUp(): Transaction reverted without a reason string
+      test/vault/VaultReentrancy.t.sol
+
+      # The test is not safe to replay because it uses impure cheatcodes: function ffi(string[] calldata commandInput) external returns (bytes memory result);
+      # TickMathTestTest.test_getTickAtSqrtRatio_matchesJavascriptImplWithin1(): EvmError: Revert
+      # TickMathTestTest.test_getSqrtRatioAtTick_matchesJavaScriptImplByOneHundrethOfABip(): EvmError: Revert
+      test/pool-cl/libraries/TickMath.t.sol
```

Comment:
```suggestion
      # https://github.com/NomicFoundation/hardhat/issues/6509
      # TokenLocker.setUp(): Transaction reverted without a reason string
      test/vault/VaultReentrancy.t.sol

      # https://github.com/NomicFoundation/hardhat/issues/6509
      # The test is not safe to replay because it uses impure cheatcodes: function ffi(string[] calldata commandInput) external returns (bytes memory result);
      # TickMathTestTest.test_getTickAtSqrtRatio_matchesJavascriptImplWithin1(): EvmError: Revert
      # TickMathTestTest.test_getSqrtRatioAtTick_matchesJavaScriptImplByOneHundrethOfABip(): EvmError: Revert
      test/pool-cl/libraries/TickMath.t.sol
```